README.md

```diff
@@ -32,7 +32,7 @@ Add this line to your application's Gemfile:
 gem 'inspec_tools'
 ```
 
-Docker installation have the Dockerfile and run `docker build -t <Container Name>`
+Docker installation have the Dockerfile and run `docker build -t <image name>`
```

Comment:
This section will be incorrect, users will be able to run the container by executing ```docker run -itv`pwd`:/share -e CHEF_LICENSE=accept mitre/inspec_tools [COMMAND]``` without having to build it locally.